mp/src/game/server/momentum/mom_system_progress.cpp

```diff
@@ -86,5 +88,25 @@ CON_COMMAND(mom_prog_reset, "Resets progress")
     g_pMomentumProgress->Reset();
 }
 
+CON_COMMAND(mom_prog_tele, "Teleports player to the last progress trigger\n")
```

Comment:
This should be inside the mom_player class, as this  class is actually an older one used for campaign progress. This command is a utility function for the player, so it makes sense in the player or timer class.

Reply:
Moved this command to mom_player.cpp but kept functionality inside. Does an additional member function inside of player class is required for such a small thing?

Reply:
Nah it's fine being in the CON_COMMAND 👍